### PR TITLE
Initial Kotlin additions

### PR DESCRIPTION
**Pull request by PeloWriter**
This WIP provides initial pieces of adding Kotlin. Note this fails during a local build.

### CODE REVIEW

**Comment by bruno-garcia** [comment]
@marandaneto when you added Dart, was there anything fancy you had to do?
build here isn't going through for some reason

File: src/platforms/kotlin/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+title: Kotlin
+caseStyle: camelCase
+supportLevel: production
+sdk: "sentry.kotlin"
```
Comment (marandaneto) [inline src/platforms/kotlin/config.yml:4]: such SDK does not exist as it falls back to `sentry.java`, should we skip it or set it as `sentry.java`?

File: src/platforms/kotlin/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+categories:
```
Comment (marandaneto) [inline src/platforms/kotlin/config.yml:5]: could be `mobile` and `server`, do we have a category for `desktop` as well?

**Comment by marandaneto** [comment]
> @marandaneto when you added Dart, was there anything fancy you had to do?
> build here isn't going through for some reason

no, but there's no kotlin platform folder nor index page, I guess that would be the bare min.

File: src/platforms/kotlin/config.yml (hunk 2 above)
Comment (marandaneto) [reply src/platforms/kotlin/config.yml:5]: yeah `desktop` also exists, so these 3 would be nice

**Comment by PeloWriter** [comment]
Closing as we fixed this with https://github.com/getsentry/sentry-docs/pull/3600

**Pull request closed without merging**